### ISSUE TITLE
Any real case use on 3d printer?

### ISSUE THREAD

**Issue by RemyBaligant** [FLYmaker/FLYF407ZG#24]
Hi,

Are you sure about the pinout? I can't get direction to work on X driver and other suspect stuff elsewhere(fans and thermistor pinout diffferent on board and marlin).
Why don't you release full schematics like other board?
It would be more easy for users.
Thanks. Best regards

![20200730_163931](https://user-images.githubusercontent.com/69038438/89026844-aeeb5680-d329-11ea-96cc-54f42ae713d7.jpg)

**Comment by FLYmaker** [comment]
![C1YTJ$E9YK(QJ88G(9 1~W4](https://user-images.githubusercontent.com/49380822/89027575-503fd000-d35d-11ea-8c85-eefd26fcd47f.png)
![0H5%3GY@0}0`{9D60(}3$_M](https://user-images.githubusercontent.com/49380822/89027578-5170fd00-d35d-11ea-8ea4-0e1e0da3231a.png)
![N(4G05}}RQ0T)QO~1K2P@SH](https://user-images.githubusercontent.com/49380822/89027582-52099380-d35d-11ea-8cfd-96a81163a809.png)
The fan pin has been repaired in my github, and today I have submitted pr to marlin

**Comment by FLYmaker** [comment]
![J}ZBTN}${XBLPH{99~2U9`P](https://user-images.githubusercontent.com/49380822/89027865-deb45180-d35d-11ea-8bb6-368c7e90da79.png)
Here is the direction of the standard plug drive
I will send the schematic to my github within 2 weeks

**Comment by RemyBaligant** [comment]
Thanks for your reply. Really good news for schematics. That was the only thing missing to make this card a best seller.
About the pinout, i mean PE0 is dir pin for X driver. But when i test my cinematics it seems that the X driver is not receiving DIR change. I tried with other tmc2209 same problem. All the rest is working so far...can't test further without being able to g28 successfully.

**Comment by FLYmaker** [comment]
Y Z  E0 Has it been tested?

**Comment by RemyBaligant** [comment]
Yes these driver are working

**Comment by FLYmaker** [comment]
![20200731193935](https://user-images.githubusercontent.com/49380822/89031784-c2b4ae00-d365-11ea-838c-770106ae6d48.png)
Check whether the voltage of this transistor is normal (compared to other drivers)
![20200731194025](https://user-images.githubusercontent.com/49380822/89031873-f4c61000-d365-11ea-90a4-40af065eafba.png)
![20200731194038](https://user-images.githubusercontent.com/49380822/89031882-f8599700-d365-11ea-9cd2-d2fc2b991dc1.png)

Check if these two points are connected

**Comment by RemyBaligant** [comment]
E4 not working. No step, no motor movement too.
I tested with ohmmeter and no continuity between the two point. I had already tried to check with stm datasheet.
So my board is defect?
Can you tell me wich pin on stm for step E4? My guess is no continuity too

**Comment by FLYmaker** [comment]
![G _5RR}$FSJ~0ZA W1B7VH](https://user-images.githubusercontent.com/49380822/89035327-52aa2600-d36d-11ea-9126-d7bd48dc218d.png)
![)F0~ %T4HDR3NAM2WPT4E{X](https://user-images.githubusercontent.com/49380822/89035337-576eda00-d36d-11ea-81d1-96c3674800c2.png)
![MNTWOQTH_ZLNF{8Z0) P$SN](https://user-images.githubusercontent.com/49380822/89035353-5ccc2480-d36d-11ea-80d2-401f7d1b426a.png)

**Comment by FLYmaker** [comment]
Your motherboard may have problems with soldering, you can find a dealer to exchange for a motherboard. If you have the ability, you can re-solder these pins with electric Luo iron

**Comment by RemyBaligant** [comment]
Yes bad soldering, i try to exchange to board. Thanks a lot for your support.
Maybe later i will have some more questions :)

**Comment by RemyBaligant** [comment]
For servo connector. Left to right i have : stm out, vcc, gnd.
Right ? 
But when i send M280 P0 S10 for example, bltouch pin won't move.
I used SERVO0_PIN PE11. Is it good pin for pwm out ?

**Comment by RemyBaligant** [comment]
![20200731_161015](https://user-images.githubusercontent.com/69038438/89043514-e4ec0300-d348-11ea-84df-e920d950acb9.jpg)

**Comment by FLYmaker** [comment]
Please use the English firmware of my github to fix the servo

**Comment by RemyBaligant** [comment]
Ok i give a try

**Comment by RemyBaligant** [comment]
![image](https://user-images.githubusercontent.com/69038438/89050557-33060400-d353-11ea-9859-dd9f882a26ec.png)

I downloaded your latest marlin release.
Wich one is good ? the pinout image or pins setting of marlin ?

**Comment by FLYmaker** [comment]
https://github.com/FLYmaker/FLYF407ZG/tree/master/English-firmware

**Comment by RemyBaligant** [comment]
I already downloaded it and it seems there is errors. Check by yourself: 
It's not the picture pinout
https://github.com/FLYmaker/FLYF407ZG/blob/master/English-firmware/Marlin/src/pins/stm32f4/pins_FLYF407ZG.h

**Comment by FLYmaker** [comment]
Sorry to update again, wait a moment

**Comment by RemyBaligant** [comment]
No problem :)

**Comment by FLYmaker** [comment]
OK, please download again

**Comment by RemyBaligant** [comment]
I received a new card from Mellow, everything running fine now. The first one had bad solderings...
And thanks for the schematics 👍 
Issue can be closed
